tests/lib/test-common.bash.in

```diff
@@ -63,6 +64,22 @@ function write_result_to_file(){
 	echo "$test_name,$test_args,$test_data,$test_platform,$os_ver" >> "$test_file"
 }
 
+# Save a test/metric result.
+# This is a wrapper function to the send_results.sh command, which ultimately decides
+# where and in what format to store or process the data.
+# Arguments:
+#  Test name
+#  Test arguments
+#  Test result
+#  Test result unit of measurement
+function save_results(){
+	if [ $# != 4 ]; then
+		die "save_results() requires 4 parameters, not $#"
+	fi
+
+	$LIB_DIR/send_results.sh -n "$1" -a "$2" -r "$3" -u "$4"
```

Comment:
I tested the PR and when calling this line I got: 

/root/go/src/github.com/01org/cc-oci-runtime/tests/metrics/density/../../lib/test-common.bash: line 80: /root/go/src/github.com/01org/cc-oci-runtime/tests/metrics/density/../../lib/send_results.sh: Permission denied

It can be fixed with: 

```
bash $LIB_DIR/send_results.sh -n "$1" -a "$2" -r "$3" -u "$4"
```